drafts/event-storming.md

```diff
@@ -0,0 +1,78 @@
+---
+title: DDD: How to do Event-Storming?
+domain: campzulu.hashnode.dev
+tags: software-architecture, ddd
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1670243050541/SwGJDMFY4.png?auto=compress
+publishAs: keerthikan
+ignorePost: true
+---
+
+# DDD: How to do Event Storming?
```

Comment:
you should not use the toplevel title with a single `#` since the blog uses the title from the metadata. You can easily have `##` titles without a top one.

Reply:
When I see the whole article, I believe you don't need tis top heading since you already have the one from the metadata.

Reply:
Good point. I didn't know that.
Thanks, I fixed it.